### PR TITLE
Fix inline bug

### PR DESCRIPTION
**Pull request by piotmag769**
Closes https://github.com/foundry-rs/starknet-foundry/issues/1591

### CODE REVIEW

**Comment by piotmag769** [comment]
@MaksymilianDemitraszek changed the abstractions, now it should be more clear

File: src/trace_reader.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ pub enum SampleType {
 
 #[allow(clippy::struct_field_names)]
 pub struct Sample {
-    pub location: Location,
+    pub locations: Vec<Location>,
```
Comment (MaksymilianDemitraszek) [inline src/trace_reader.rs:31]: Is it a correct name? is it not callstack or path?

Current name suggested that this sample was collected from multiple places.

File: src/trace_reader.rs [hunk 2]
```diff
@@ -13,12 +13,12 @@ use crate::trace_data::{CallTrace, DeprecatedSyscallSelector, ExecutionResources
 pub struct FunctionName(pub String);
 
 #[derive(Clone, Hash, Eq, PartialEq)]
-pub struct Location(pub Vec<FunctionName>);
+pub struct Location(pub FunctionName);
```
Comment (MaksymilianDemitraszek) [inline src/trace_reader.rs:16]: Is there a point for this newtype to still exist now?

**Review by MaksymilianDemitraszek — APPROVED** [review]
Please address the comments

**Pull request merged**